core/lib/network-request.js

```diff
@@ -320,12 +318,16 @@ class NetworkRequest {
   }
 
   /**
-   * @param {string=} sessionId
+   * @param {string|undefined} sessionId
    */
   setSession(sessionId) {
     this.sessionId = sessionId;
   }
 
+  get oopif() {
```

Comment:
`isOopif` to match e.g. `isLinkPreload`?

Any reason to make it a getter and not a function?

Reply:
Mostly I didn't want to convert some of tests that use raw network objects in this PR.

Reply:
> isLinkPreload

Oof, I assumed this was a method, so my previous comment didn't make much sense.

I think having two properties here that need to sync up will be easy to mess up in a test. Renaming to `isOutOfProcessIframe` seems good though.